src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -128,15 +132,55 @@ public IAIState getStateAfterPickUp()
 
     /**
      * State to pick up material before going back to work.
+     *
      * @return the next state to go to.
      */
     public IAIState pickUpMaterial()
     {
         final BuildingBuilder building = getOwnBuilding();
-        final List<Predicate<ItemStack>> neededItemsList = new ArrayList<>();
-        for (final BuildingBuilderResource stack : building.getNeededResources().values())
+        final List<Tuple<Predicate<ItemStack>, Integer>> neededItemsList = new ArrayList<>();
+        boolean ONLY_64 = false;
```

Comment:
This is not a constant. Adapt formatting.

Reply:
changing